packages/backend/src/peripherals/database/VerifierEventRepository.ts

```diff
@@ -47,13 +47,30 @@ export class VerifierEventRepository
   async getAll() {
     const rows = await this.knex('verifier_events').select('*')
     this.logger.debug({ method: 'getAll', rows: rows.length })
+
+    return rows.map(toRecord)
+  }
+
+  async getAllAfter(blockNumber: number) {
+    const rows = await this.knex('verifier_events')
+      .where('blockNumber', '>', blockNumber)
```

Comment:
Should this be inclusive or exclusive @sz-piotr ?

Reply:
It was supposed to be `getAllBefore` and exclusive. But I'm not sure if we need this, see comment in VerifierCollector